code/src/cljs/sixsq/nuvla/ui/pages/edges/views_timeseries.cljs

```diff
@@ -53,13 +53,16 @@
       [ui/CardMeta {:style {:font-size "tiny"}}
        (str "on " (time/time->format (:timestamp (first ts-data))))]]
      [ui/CardContent
-      [ui/CardDescription
+      [ui/CardDescription {:style {:overflow-x "auto"}}
        (if (seq least-available-edge)
-         (into [ui/Table {:basic "very"}
+         (into [ui/Table {:basic "very"
+                          :unstackable true}
                 [ui/TableHeader
                  [ui/TableRow
                   [ui/TableHeaderCell (str/capitalize (@tr [:name]))]
-                  [ui/TableHeaderCell {:textAlign "right"} (str (@tr [:availability]) " (%)")]]]]
+                  [ui/TableHeaderCell {:textAlign "right"
+                                       :style {:white-space "nowrap"}}
+                   (str (@tr [:availability]) " (%)")]]]]
                (mapv (fn [bucket]
```

Comment:
Minor comment. I'll push and merge it. 
The table row should be in a table body. And instead of into + mapv use a for loop.

Reply:
`for` is lazy though, so I think it's not so well-suited when returning lists in the frontend.